src/genai/file/__init__.py

```diff
@@ -1,4 +1,12 @@
 """Modules containing functionalities related to static assets (user's files)"""
-
 from genai.file.file_service import *
-from genai.file.schema import *
+
+
+def __getattr__(name: str):
```

Comment:
you can technically import everything from schema using
`from genai.file import ...`, but you'll get a clear deprecation warning